source/tutorials/migrate-time-series.txt

```diff
@@ -1,198 +1,319 @@
+.. _kafka-tutorial-migrate-time-series:
 
 ==========================================================
 Migrate an Existing Collection to a Time Series Collection
 ==========================================================
 
-.. default-domain:: mongodb
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
 
-In this tutorial, you can learn how to convert an existing MongoDB
+Follow this tutorial to learn how to convert an existing MongoDB
 collection to a **time series collection** using the {+mkc+}.
 
 Time series collections efficiently store sequences of measurements
 over a period of time. Time series data consists of measurement data collected
 over time, metadata that describes the measurement, and the time of the
 measurement.
 
-You can configure the source connector to read your existing MongoDB
-collection and the sink connector to write Kafka topic data into a MongoDB
-time series collection.
+To convert data from a MongoDB collection to a time series collection using
+the connector, you need to perform the following tasks:
 
-To learn more about MongoDB time series collections, see the MongoDB
-manual page on :manual:`Time Series Collections </core/timeseries-collections/>`.
+#. Identify the time field common to all documents in the collection.
+#. Configure a source connector to copy the existing collection data to a
+   Kafka topic.
+#. Configure a sink connector to copy the Kafka topic data to the time series
+   collection.
 
-Scenario
---------
+In this tutorial, you perform these preceding tasks to migrate stock data
+from a collection to a time series collection. The time series collection
+stores the data more efficiently and retains the ability to analyze stock
+performance over time using aggregation operators.
 
-Suppose you accumulated stock price data in a MongoDB collection and have
-the following needs:
+.. important:: Apple M1 incompatibility
 
-- More efficient storage of the price data
-- Maintain the ability to analyze stock performance over time using
-  aggregation operators
+   This tutorial requires the time series collection feature introduced in
+   MongoDB version 5.0. The MongoDB Docker image does not currently build for
+   any version after 4.x on an M1 CPU, and therefore cannot run the complete
+   tutorial. MongoDB developers are currently working on resolving this
+   issue.
 
-After reading about MongoDB time series collections, you decide to migrate
-your existing collection into a time series one. Learn how to perform this
-migration in the following sections.
+.. include:: /includes/tutorials/setup.rst
 
-Steps to Migrate to a Time Series Collection
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Migrate a Collection to a Time Series Collection
+------------------------------------------------
 
-To migrate an existing MongoDB collection to a time series collection, you
-need to perform the following tasks:
+.. procedure::
+   :style: connected
```

Comment:
Good idea -- I'll create a task to address this since the scope is across tutorials. Will need to do some investigation to see if it looks easier to follow or more difficult to follow since duplicate numbering (substep numbers contain the same numbers as the step they reside in) could be confusing especially when there's a lot of vertical space between the substep and its parent.

Reply:
https://jira.mongodb.org/browse/DOCSP-22929